Fix maintenance end-to-end failing due to invalid Link on creation of circuits

Since the introduction of PR kytos-ng/mef_eline#78 (which validates if the requested Path is actually a valid Path), the end-to-end tests fro maintenance are failing. Example:
```
_______ TestE2EMaintenance.test_010_create_mw_on_switch_should_move_evc ________

self = <tests.test_e2e_15_maintenance.TestE2EMaintenance instance at 0x7f8e9ef7e2d8>

   def test_010_create_mw_on_switch_should_move_evc(self):
       """Tests the EVC behaviors during maintenance
       Test:
           /api/kytos/maintenance on POST
       """

       self.restart_and_create_circuit()

       # Sets up the maintenance window information
       mw_start_delay = 60
       mw_duration = 60
       start = datetime.now() + timedelta(seconds=mw_start_delay)
       end = start + timedelta(seconds=mw_duration)

       # Sets up the maintenance window data
       payload = {
           "description": "mw for test 010",
           "start": start.strftime(TIME_FMT),
           "end": end.strftime(TIME_FMT),
           "items": [
               "00:00:00:00:00:00:00:02"
           ]
       }

       # Creates a new maintenance window
       api_url = KYTOS_API + '/maintenance'
       response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
       assert response.status_code == 201
       data = response.json()
       assert 'mw_id' in data

       # Waits for the MW to start
       time.sleep(mw_start_delay + 10)

       # Switch 1 and 3 should have 3 flows; Switch 2 should have only 1 flow.
       s1, s2, s3 = self.net.net.get('s1', 's2', 's3')
       flows_s1 = s1.dpctl('dump-flows')
       flows_s2 = s2.dpctl('dump-flows')
       flows_s3 = s3.dpctl('dump-flows')
     assert len(flows_s1.split('\r\n ')) == 3
E       AssertionError: assert 1 == 3
E        +  where 1 = len([' cookie=0x0, duration=90.517s, table=0, n_packets=55, n_bytes=2310, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'])
E        +    where [' cookie=0x0, duration=90.517s, table=0, n_packets=55, n_bytes=2310, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'] = <built-in method split of str object at 0x55c75adfb340>('\r\n ')
E        +      where <built-in method split of str object at 0x55c75adfb340> = ' cookie=0x0, duration=90.517s, table=0, n_packets=55, n_bytes=2310, priority=1000,dl_vlan=3799,dl_type=0x88cc actions=CONTROLLER:65535\r\n'.split

tests/test_e2e_15_maintenance.py:117: AssertionError
_____________ TestE2EMaintenance.test_085_end_running_mw_on_switch _____________

self = <tests.test_e2e_15_maintenance.TestE2EMaintenance instance at 0x7f8e9f03d710>

   def test_085_end_running_mw_on_switch(self):
       self.restart_and_create_circuit()

       # Sets up the maintenance window information
       mw_start_delay = 30
       mw_duration = 60
       start = datetime.now() + timedelta(seconds=mw_start_delay)
       end = start + timedelta(seconds=mw_duration)

       # Sets up the maintenance window data
       payload = {
           "description": "mw for test 85",
           "start": start.strftime(TIME_FMT),
           "end": end.strftime(TIME_FMT),
           "items": [
               "00:00:00:00:00:00:00:02"
           ]
       }

       # Creates a new maintenance window
       api_url = KYTOS_API + '/maintenance'
       response = requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
       data = response.json()

       # Extracts the maintenance window id from the JSON structure
       mw_id = data["mw_id"]

       # Gets the maintenance schema
       api_url = KYTOS_API + '/maintenance/' + mw_id
       response = requests.get(api_url)
       assert response.status_code == 200
       json_data = response.json()
       assert json_data['id'] == mw_id

       # Waits for the MW to start
       time.sleep(mw_start_delay + 5)

       # Verifies the flow behavior during the maintenance
       s2 = self.net.net.get('s2')
       flows_s2 = s2.dpctl('dump-flows')
       assert 'dl_vlan=100' not in flows_s2
       assert len(flows_s2.split('\r\n ')) == 1

       # Checks connectivity during maintenance
       h11, h3 = self.net.net.get('h11', 'h3')
       h11.cmd('ip link add link %s name vlan100 type vlan id 100' % (h11.intfNames()[0]))
       h11.cmd('ip link set up vlan100')
       h11.cmd('ip addr add 100.0.0.11/24 dev vlan100')
       h3.cmd('ip link add link %s name vlan100 type vlan id 100' % (h3.intfNames()[0]))
       h3.cmd('ip link set up vlan100')
       h3.cmd('ip addr add 100.0.0.2/24 dev vlan100')
       result = h11.cmd('ping -c1 100.0.0.2')
     assert ', 0% packet loss,' in result
E       AssertionError: assert ', 0% packet loss,' in 'PING 100.0.0.2 (100.0.0.2) 56(84) bytes of data.\r\nFrom 100.0.0.11 icmp_seq=1 Destination Host Unreachable\r\n\r\n--- 100.0.0.2 ping statistics ---\r\n1 packets transmitted, 0 received, +1 errors, 100% packet loss, time 0ms\r\n\r\n'

tests/test_e2e_15_maintenance.py:870: AssertionError

```

Eventually, more tests fail.

Investigating this situation, it seems to be related to the fact that EVC is not actually being created due to invalid link:
```
# grep 'POST.*mef_eline/v2/evc' /var/log/syslog
Dec 21 18:54:57 35d0ab0b801d 2021-12-21 18:54:57,700 (Thread-21) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 18:54:57] "#033[31m#033[1mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 400 -
Dec 21 19:00:39 35d0ab0b801d 2021-12-21 19:00:39,343 (Thread-24) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:00:39] "#033[31m#033[1mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 400 -
Dec 21 19:01:14 35d0ab0b801d 2021-12-21 19:01:14,938 (Thread-25) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:01:14] "#033[31m#033[1mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 400 -
Dec 21 19:08:16 35d0ab0b801d 2021-12-21 19:08:16,476 (Thread-62) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:08:16] "#033[37mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 201 -
Dec 21 19:11:45 35d0ab0b801d 2021-12-21 19:11:45,948 (Thread-33) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:11:45] "#033[37mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 201 -
Dec 21 19:15:26 35d0ab0b801d 2021-12-21 19:15:26,236 (Thread-30) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:15:26] "#033[37mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 201 -
Dec 21 19:18:51 35d0ab0b801d 2021-12-21 19:18:51,096 (Thread-33) werkzeug:INFO _internal:113:  127.0.0.1 - - [21/Dec/2021 19:18:51] "#033[37mPOST /api/kytos/mef_eline/v2/evc/ HTTP/1.1#033[0m" 201 -

# from a simulated request with more logging enabled
{"code":400,"description":"primary_path is not valid: Link Link(Interface('s1-eth3', 3, Switch('00:00:00:00:00:00:00:01')), Interface('s2-eth2', 2, Switch('00:00:00:00:00:00:00:02'))) is not available.","name":"Bad Request"}
```

As we can see from actually logs, some of the requests returned error 400.

This is likely to be happening because the of_lldp takes a couple of seconds to discovery the topology. We usually wait 10 seconds before create the EVC in the end-to-end tests. [Maintenance end-to-end tests are waiting just 5 seconds](https://github.com/amlight/kytos-end-to-end-tests/blob/master/tests/test_e2e_15_maintenance.py#L61).

--- a/tests/test_e2e_15_maintenance.py
+++ b/tests/test_e2e_15_maintenance.py
@@ -54,13 +54,15 @@ class TestE2EMaintenance:
             ],
         }
         api_url = KYTOS_API + '/mef_eline/v2/evc/'
-        requests.post(api_url, data=json.dumps(payload), headers={'Content-type': 'application/json'})
+        response = requests.post(api_url, json=payload)
+        assert response.status_code == 201, response.text
+
 
     def restart_and_create_circuit(self):
         self.net.restart_kytos_clean()
-        time.sleep(5)
+        time.sleep(10)
         self.create_circuit(100)
-        time.sleep(20)
+        time.sleep(10)
 
     def test_005_list_mw_should_be_empty(self):
         """Tests if the maintenances list is empty at the beginning

